src/components/DateInput/__stories__/Controlled.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { Story } from '@storybook/react'
+import { ComponentProps, useState } from 'react'
+import DateInput from '..'
+
+export const Controlled: Story<ComponentProps<typeof DateInput>> = () => {
+  const [value, setValue] = useState<Date | null>(
+    new Date('December 17, 1995 03:24:00'),
+  )
+
+  return <DateInput label="Date" value={value as Date} onChange={setValue} />
+}
```

Comment:
If we spread props here won't it remove `() =>` in code snippet of storybook?

Reply:
Yes, but it will also remove the `useState` and such the `value/setValue` pair:

<img width="456" alt="Screenshot 2022-10-27 at 10 39 47" src="https://user-images.githubusercontent.com/43268759/198235481-6ba02e6f-a6ba-4025-8e47-bfb23299afc5.png">

IMO it's better to keep it like this.

Reply:
Okay then let's keep it like that 😄